### DISCUSSION TITLE
我因为证书问题而无法安装

### DISCUSSION THREAD

**Discussion by AllayFocalors** [Richasy/Bili.Copilot#863] [Q&A]
我运行了安装包中BiliCopilot.UI_2.2410.9.0_x64.msix，弹出安装界面，却说：无法验证此应用包的发布者证书。请与系统管理员或应用开发人员联系，以获取具有验证证书的新应用包。必须验证应用包中签名的根证书和所有直接证书(0x800B010A)
![Screenshot 2024-07-21 181437](https://github.com/user-attachments/assets/173c6ac5-599f-49d3-8347-a7d793225fc4)
请问我要怎么做才能正常安装

**Comment by sunnysky123** [comment]
若无法通过文件安装，也可通过微软商店安装，项目的readme文档里有商店的链接
